### ISSUE TITLE
Document RTD workflow issues

### ISSUE THREAD

**Issue by dwf** [mila-udem/blocks#865]
The recent flurry of issue tracker activity regarding RTD's various and bizarre problems makes me wonder whether we need documentation about how to fix problems with RTD. Some explicitly narrative docs about whatever has been done recently may prove useful for posterity, more useful and expedient than combing through old conversations on the issue tracker that span multiple tickets and PRs.

**Comment by dmitriy-serdyuk** [comment]
Very good idea! It might be difficult since RTD API may change in the future and the documentation will not be up-to-date.

**Comment by dwf** [comment]
Yes, RTD may change, and that's a problem we will have to deal with when it comes. I'm more worried about us breaking stuff on our end though. :smile:
